mountpoint-s3-fs/src/superblock.rs

```diff
@@ -1450,16 +1451,14 @@ impl InodeMap {
 }
 
 #[derive(Debug, Default)]
+/// A wrapper around a `HashMap<InodeNo, u32>` to manage reader counts for Inodes
+/// Ensures that the inodes are locked for writing while performing these operations.
```

Comment:
The fact that it wraps a hash map is not too interesting (and visible 2 lines below). I'd rather mention that it only stores entries with > 0 readers.